server/src/test/java/org/elasticsearch/cluster/metadata/InferenceFieldMetadataTests.java

```diff
@@ -58,18 +55,56 @@ protected boolean supportsUnknownFields() {
         return true;
     }
 
+    @Override
+    protected Predicate<String> getRandomFieldsExcludeFilter() {
```

Comment:
@Mikep86 Per your suggestion, this is failing: 
```
java.lang.AssertionError: expected:<{"eVtz":{"inference_id":"ksumkinbbyn","search_inference_id":"cqhvmbgyoigq","source_fields":["QdaMbXfcDi","yZUcUWObnq","TCWLZVfbXL","fdStjoAcwM","ltWaXmNfAF"],"chunking_settings":{"strategy":"sentence_boundary","sentence_overlap":1,"max_chunk_size":33}}}> but was:<{"eVtz":{"inference_id":"ksumkinbbyn","search_inference_id":"cqhvmbgyoigq","source_fields":["QdaMbXfcDi","yZUcUWObnq","TCWLZVfbXL","fdStjoAcwM","ltWaXmNfAF"],"chunking_settings":{"max_chunk_size":33,"tRzqUWIxcJ":[0.12235856,0.35687828,0.88086027],"strategy":"sentence_boundary","sentence_overlap":1}}}
```
That is why I originally set `supportUnknownFields()` - happy to discuss further though